site/content/docs/main/restore-hooks.md

```diff
@@ -154,4 +154,106 @@ and
 }
 ```
 
+## Exec Restore Hooks
```

Comment:
I can't leave a comment further up the file, and I didn't remove it when resolving the merge conflicts on this branch, but we can remove line 11 now.

Reply:
we can get this in and fix that in a follow-up PR. WDYT?

Reply:
Yeah, I'm fine with that, mostly just wanted to make a note of it.

Reply:
Created #2919